components/hab/src/cli.rs

```diff
@@ -526,6 +526,8 @@ pub fn get(feature_flags: FeatureFlag) -> App<'static, 'static> {
                 (@arg FORCE: --force "Skips checking availability of package and \
                     force uploads, potentially overwriting a stored copy of a package. \
                     (default: false)")
+                (@arg DISABLE_BUILD: --("disable-build")  "Do not create a build group on package upload. This flag has no \
```

Comment:
@apriofrost may want to wordsmith, but I might just keep the phrasing simple and say "Do not auto build on package upload"

Reply:
Agree with @chefsalim . I'd suggest "Disable auto-build for all packages in this upload." to align with the language we use in the Builder UI.